src/modules/auto-invisible-recaptcha.js

```diff
@@ -44,6 +45,7 @@
 			var wid = window.grecaptcha.render(id, {
 				sitekey: t.attr('data-sitekey'),
 				badge: t.attr('data-badge') || 'bottomright',
+				hl: html.attr('lang'),
```

Comment:
Maybe set a default value for the case lang attribute is not present

Reply:
If the value is invalid or missing, Google's fallback will be the browser detection.